### PR TITLE
Pango markup config

### PR DESCRIPTION
**Pull request by Hummer12007**
This PR adds the ability to toggle parsing pango markup for plaintext statuslines.

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Can you call the option "pango_markup"?

File: swaybar/config.c [hunk 1]
```diff
@@ -46,6 +46,7 @@ char *parse_font(const char *font) {
 struct config *init_config() {
 	struct config *config = calloc(1, sizeof(struct config));
 	config->status_command = NULL;
+	config->plaintext_markup = true;
```
Comment (mikkeloscar) [inline swaybar/config.c]: I might be missing how this works compared to i3, but as I understand this is not possible in i3, so maybe it should default to `false` so it behaves as expected by default.

Comment (SirCmpwn) [inline swaybar/config.c]: The default text rendering in i3 is with pango markup enabled.

**Comment by Hummer12007** [comment]
Config options must be somehow renamed, too, in order to make sense though.

**Pull request merged**